charts/monitoring-config/rules/signon.yaml

```diff
@@ -12,3 +12,4 @@
         summary: Signon API User token is due to expire soon
         description: >-
           A token is about to expire within the next two weeks and needs rotating.
+        runbook: https://docs.publishing.service.gov.uk/manual/alerts/signon-api-user-token-expires-soon.html
```

Comment:
Nit: runbook_url is more explicit as what the field is

Reply:
Oh yeah, good idea.